app/screens/sidebar/cityFilter/CityFilterContainer.js

```diff
@@ -16,6 +17,16 @@ export function UnconnectedCityFilterContainer(props) {
 
 UnconnectedCityFilterContainer.propTypes = {
   cities: PropTypes.arrayOf(PropTypes.string).isRequired,
+  onSelect: PropTypes.func.isRequired,
+  selected: PropTypes.string.isRequired,
 };
 
-export default connect(selector)(UnconnectedCityFilterContainer);
+const actions = {
+  onSelect: (event) => {
+    const target = event.target;
+    const value = target.options[target.selectedIndex].value;
```

Comment:
- [ ] Is there any reason for not just using `const value = event.target.value;`